components/item/ItemHero.vue

```diff
@@ -21,7 +21,13 @@
           >
             {{ rightsStatement }}
           </span>
-          <div>
+          <div class="d-flex">
+            <UserButtons
+              v-if="showUserButtons"
+              v-model="identifier"
+              @like="$emit('like', identifier)"
```

Comment:
These two event emissions are redundant, I think. Remove them if you concur.

Reply:
I agree, removed